### PR TITLE
Implement __str__ and field accessors for revision ranges.

### PR DESCRIPTION
**Pull request by boehmseb**
I need this for some work over at the VaRA-Tool-Suite.

### CODE REVIEW

**Comment by boehmseb** [comment]
Well, I messed up a rebase here...

File: benchbuild/utils/revision_ranges.py [hunk 1]
```diff
@@ -93,12 +93,19 @@ def __init__(self, rev_id: str, comment: tp.Optional[str] = None):
...
+        return self.__id
+
     def init_cache(self, repo_path: str) -> None:
         pass
 
     def __iter__(self) -> tp.Iterator[str]:
         return [self.__id].__iter__()
 
+    def __str__(self) -> str:
+        return self.__id
```
Comment (vulder) [inline benchbuild/utils/revision_ranges.py:107]:
```suggestion
        return self.rev_id
```
Using the defined property

**Review by vulder — APPROVED** [review]
@simbuerg Do you/BB document properties?

**Comment by simbuerg** [comment]
No need for simple attributes, the code should be sufficient as documentation.

How did you nuke mypy on the CI build? :D

**Review by simbuerg — COMMENTED** [review]
LGTM. Not sure why mypy fails on this commit.

**Pull request merged**